lib/openxml/elements/paragraph.rb

```diff
@@ -0,0 +1,8 @@
+module OpenXml
+  module Elements
+    class Paragraph < OpenXml::Container
+      namespace :a
```

Comment:
Why don't we merge this one in, and then make a subsequent pass that moves all this to the drawingML lib?